pkgs/development/libraries/libprotobuf-mutator/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, clangStdenv
+, cmake
+, fetchFromGitHub
+, git
+, libtool
+, lzma
+, ninja
+, pkg-config
+, protobuf
+, zlib
+}:
+
+let
+  # See `src/cmake/external/googletest.cmake`.
+  googletest_src = fetchFromGitHub {
+    owner = "google";
+    repo = "googletest";
+    rev = "release-1.11.0";
+    sha256 = "sha256-SjlJxushfry13RGA7BCjYC9oZqV4z6x8dOiHfl/wpF0=";
+  };
+in
+clangStdenv.mkDerivation rec {
+  pname = "libprotobuf-mutator";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-E4EL0c/v1IAZ7nDQQFRflls5BG5juUfq0ztTBuAGDXk=";
+  };
+
+  patches = [ ./local_googletest.patch ];
+
+  postPatch = ''
+    # Remove examples as they require to pull external repositories.
+    rm -R examples/
```

Comment:
```suggestion
    rm -r examples/
```

Reply:
These are an alias to each other.
Why is that a suggested change? This feels beyond nitpicking...

Reply:
if you ripgrep nixpkgs... 

 ```
rg "rm " | grep "\-r" | wc -l 
1098

rg "rm " | grep "\-R" | wc -l
38

```

Reply:
Habit. If you read the darwin man page -R was actually deprecated at some point but it is no longer.

Reply:
i was talking about solaris. ;-)